rules/windows/persistence_runtime_run_key_startup_susp_procs.toml

```diff
@@ -0,0 +1,56 @@
+[metadata]
+creation_date = "2020/11/19"
+maturity = "production"
+updated_date = "2020/11/19"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies execution of suspicious persistent programs (scripts, rundll32, etc.) by looking at process lineage and
+command line usage.  
+"""
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Execution of Persistent Suspicious Program"
+risk_score = 47
+rule_id = "e7125cea-9fe1-42a5-9a05-b0792cf86f5a"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Persistence"]
+type = "eql"
+
+query = '''
+/* userinit followed by explorer followed by early child process of explorer (unlikely to be launched interactively) within 1m */
+sequence by host.id, user.name with maxspan = 2m
+  [process where event.type in ("start", "process_started") and process.name : "userinit.exe" and process.parent.name : "winlogon.exe"]
+  [process where event.type in ("start", "process_started") and process.name : "explorer.exe"]
+  [process where event.type in ("start", "process_started") and process.parent.name : "explorer.exe" and
+   /* add suspicious programs here */
+   process.pe.original_file_name in ("cscript.exe",
+                                     "wscript.exe",
+                                     "PowerShell.EXE",
+                                     "Cmd.Exe",
```

Comment:
What about logon scripts? The `cmd.exe` seems very noisy

Reply:
removed